maestro/server/deploy/templates/postgres-breakglass.deployment.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: postgres-breakglass
+  namespace: '{{ .Release.Namespace }}'
+  labels:
+    app: postgres-breakglass
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      app: postgres-breakglass
+  template:
+    metadata:
+      labels:
+        app: postgres-breakglass
+        azure.workload.identity/use: "true"
+    spec:
+      serviceAccount: '{{ .Values.maestro.serviceAccount }}'
+      serviceAccountName: '{{ .Values.maestro.serviceAccount }}'
+      initContainers:
+      - name: init
+        image: 'mcr.microsoft.com/azure-cli:cbl-mariner2.0'
+        imagePullPolicy: IfNotPresent
+        command:
+        - /bin/sh
+        - -c
+        - |
+          echo "PostgreSQL breakglass init container started at $(date)"
+          echo "Logging into Azure using workload identity..."
+          az login --federated-token "$(cat $AZURE_FEDERATED_TOKEN_FILE)" --service-principal -u $AZURE_CLIENT_ID -t $AZURE_TENANT_ID
+
+          echo "Getting PostgreSQL access token..."
+          ACCESS_TOKEN=$(az account get-access-token --resource-type oss-rdbms -o json | jq .accessToken -r)
+
+          if [ -z "$ACCESS_TOKEN" ] || [ "$ACCESS_TOKEN" = "null" ]; then
+            echo "Failed to get access token"
+            exit 1
+          fi
+
+          echo "Writing access token and connect script to shared volume..."
+          echo "$ACCESS_TOKEN" > /breakglass/postgres_token
```

Comment:
I don't see this file being used anywhere?

Reply:
good spot. this was my first solution and i was unsure if i should keep it around for something convenient that might come up. not worth the effort. i removed it